thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/template.hbs

```diff
@@ -0,0 +1,86 @@
+{{#if anomaly}}
+  {{#if isNotResolved}}
```

Comment:
I know this is a mvp, but shouldn't the feedback component always be there if an anomaly exist? 

(the case where a user wants to change a false alarm to a true anomaly)

Reply:
fixed